FEATURES.md

```diff
@@ -117,7 +117,7 @@ If your PR makes notable changes to Winit's features, please update this section
 * Setting the taskbar icon
 * Setting the parent window
 * `WS_EX_NOREDIRECTIONBITMAP` support
-* Theme the title bar according to Windows 10 Dark Mode setting
+* Theme the title bar according to Windows 10 Dark Mode setting or force a theme
```

Comment:
I don't like `a`, since you're forcing some specific theme and not current one, etc. If you have other ideas on how to show the intent more clear, let me know.

```suggestion
* Theme the title bar according to Windows 10 Dark Mode setting or force preferred theme
```